src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-uk.gov.companieshouse.chdorderapi.health=/healthcheck
\ No newline at end of file
+uk.gov.companieshouse.chdorderapi.health=/chd-order-api/healthcheck
```

Comment:
Good job on fixing this, another story was created to fix the issue with the healthcheck, that can be resolved.